src/test/java/core/basesyntax/ArrayListTest.java

```diff
@@ -9,7 +9,7 @@ public class ArrayListTest {
 
     @Test
     public void addElementToArrayList() {
-        ArrayList<String> arrayList = new ArrayList<>();
+        ArrayList<String> arrayList = new ArrayList<>(size);
```

Comment:
It seems you are using a variable 'size' that has not been declared or initialized. You should replace it with a constant representing the initial capacity of the ArrayList or remove the parameter if you have a default constructor.